static/_redirects

```diff
@@ -30,3 +28,5 @@
 
 /docs/contributing/*                            /contributing/:splat                                301!
 /docs/*                                         /flux/:splat                                        301!
+
+/legacy/*                                       /flux/migration                                     301!
```

Comment:
We could theoretically redirect to GitHub, but that would require a bit of special casing... any strong opinions here?

Reply:
I would go with the redirect to migration.

Reply:
Thanks for the feedback!